server/svix-server/src/core/kv_backend/none.rs

```diff
@@ -5,43 +5,38 @@ use std::time::Duration;
 
 use axum::async_trait;
 
-use super::{Cache, CacheBehavior, CacheKey, CacheValue, Result, StringCacheValue};
-
-pub fn new() -> Cache {
-    tracing::warn!("Running with caching disabled will negatively affect performance. Idempotency is not supported without a cache.");
-    NoCache {}.into()
-}
+use super::{KeyValueStoreBackend, Result, StringValue, Value};
 
 #[derive(Clone)]
-pub struct NoCache;
+pub struct NoKeyValueStore;
```

Comment:
If you are already renaming it, please rename it to KeyValueStoreNone to be consistent with the rest.